ledger/eval/eval.go

```diff
@@ -129,6 +132,15 @@ type roundCowBase struct {
 	// The account data store here is always the account data without the rewards.
 	accounts map[basics.Address]ledgercore.AccountData
 
+	// The online accounts that we've already accessed during this round evaluation. This is a
+	// cache used to avoid looking up the same account data more than once during a single evaluator
+	// execution. The OnlineAccountData is historical and therefore won't be changing.
+	voters map[basics.Address]basics.OnlineAccountData
```

Comment:
nit: `voters` is used for stateproof here in ledger package. maybe use some other name to decrease cognitive load?

Reply:
I guess I'll go with `onlineAccounts` here.  Are you concerned about the opcode name `voter_params_get` as well?

Reply:
no concerns on the opcode name